libr/anal/p/anal_x86_cs.c

```diff
@@ -53,6 +54,8 @@ call = 4
 #define ARG1_AR      1
 #define ARG2_AR      2
 
+static RRegItem base_regs[2];
```

Comment:
global vars are bad, can we get rid of them?

Reply:
I may make them static within a function and pass as arguments when other function needs them, if it's better.

Reply:
Better move them into RAnal.. but we should have a special struct to hold all those vars, and then put it inside RAnal. Statics inside functions are also globals